cg/services/orders/submitters/order_submitter_registry.py

```diff
@@ -80,18 +80,6 @@ def get_order_submitter(self, order_type: OrderType) -> OrderSubmitter:
         StoreCaseOrderService,
         CaseOrderSubmitter,
     ),
-    OrderType.BALSAMIC_QC: (
-        OrderLimsService,
-        ValidateCaseOrderService,
-        StoreCaseOrderService,
-        CaseOrderSubmitter,
-    ),
-    OrderType.BALSAMIC_UMI: (
```

Comment:
I think we are keeping Balsamic-UMI around!

Reply:
Yes, that one is frequently used!

Reply:
Removed by mistake. Restored!